### PR TITLE
Updates for Katello 4.9.0.rc1

### PR DESCRIPTION
**Pull request by ianballou**
Katello 4.9.0!

### CODE REVIEW

File: guides/doc-Release_Notes/topics/katello.adoc [hunk 1]
```diff
@@ -4,8 +4,32 @@
 [id="katello-headline-features"]
 == Headline Features
 
-[id="katello-upgrade-warnings"]
-== Upgrade Warnings
+=== Custom repository sets switching to disabled-by-default
+Custom products are now disabled by default in Katello. During upgrade to Katello 4.9, content overrides will be automatically added so that existing hosts and activation keys have access to the same repositories before and after upgrade. Going forward, newly created custom products will be disabled by default.
```
Comment (mjivraja) [inline guides/doc-Release_Notes/topics/katello.adoc:8]: Can we follow one sentence per line here? @maximiliankolb @Lennonka

Comment (Lennonka) [reply guides/doc-Release_Notes/topics/katello.adoc:8]: Ideally yes.

File: guides/doc-Release_Notes/topics/katello.adoc [hunk 2]
```diff
@@ -4,8 +4,32 @@
...
+=== Content view publish recommendations
+Katello now tells users if a content view publish is unnecessary based on changes to repositories and filters. Likewise, new icons in the UI will communicate if a new version should be published to receive new updates.
+
+=== Content view version filtering history
+Content view versions that had filters applied will receive a new flag in the UI. Filters that were applied to previous content view versions are now available to fetch from the API and Hammer.
+
+=== Incremental update dependency solving option in UI
+Dependency solving during incremental update via the UI is now optional. A checkbox with some information will allow the user to disable it for a potential publishing speed increase.
+
+=== Repositories sets now filterable by type
```
[the remark this answers is not in the document]
Comment (Lennonka) [reply guides/doc-Release_Notes/topics/katello.adoc:28]:
```suggestion
=== Repository sets now filterable by type
```

File: guides/doc-Release_Notes/topics/katello.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/doc-Release_Notes/topics/katello.adoc:8]: Yes, it's always a good idea. BUT I don't use this in downstream yet & it's not within `guides/common/`, so I am really OK either way. If the text comes from, say, git commit messages, then I think we should not force anyone to rewrite it.

Comment (mjivraja) [reply guides/doc-Release_Notes/topics/katello.adoc]: Ack!

**Comment by ianballou** [comment]
I have merge access (recently added) but I'm not sure what the docs policy is -- is this good to go in?

**Comment by Lennonka** [comment]
I think you can go ahead and merge it.

**Pull request merged**